src/Lecture1.hs

```diff
@@ -160,5 +145,10 @@ lowerAndGreater n list =
     show lts ++ 
     " elements"
         where
-            gls = length $ filter (< n) list
-            lts = length $ filter (> n) list
+            (gls, lts) = foldl (
+                \(a, b) -> 
+                    \x -> 
+                        if x < n then (a + 1, b) 
+                        else if x > n then (a, b + 1)
+                        else (a, b)
+            ) (0, 0) list
```

Comment:
GHC still produces a compilation error. Not sure why but try indenting the `)` a little bit further
```suggestion
                ) (0, 0) list
```